### DISCUSSION TITLE
Open_Shell is not displayed in German

### DISCUSSION THREAD

**Discussion by stardepp** [Open-Shell/Open-Shell-Menu#1065] [Q&A] [answered]
Done. I figured it out myself.

**Comment by blackcrack** [comment, marked as answer]
Hi,

Translations:
https://coddec.github.io/Classic-Shell/www.classicshell.net/translations/index.html

scrolling down up to:
```
You can also manually download and install the update. These are the files available for the latest version 4.2.4:   
```

 Download all the nice *.DLL from this page
kick in the Program folder of OpenShellMenu and restart the Program..

best


so, why did you not close this tread.. by side of answer, should be the "answer&close" button or something.. i have not often post in discussions and it is in possible to be i am wrong..
so, if possible, then close this discussion..
thanks :)
